int32 homescore
int32 awayscore
bool play
bool swapsides
